src/aiida/orm/nodes/data/code/legacy.py

```diff
@@ -231,7 +231,7 @@ def get_code_helper(cls, label, machinename=None, backend=None):
             return result[0]
 
     @classmethod
-    def get(cls, pk=None, label=None, machinename=None):
+    def get(cls, pk=None, label=None, machinename=None):  # type: ignore[override]
```

Comment:
I don't understand why mypy complains here:

```
src/aiida/orm/nodes/data/code/legacy.py:234: error: Signature of "get"
incompatible with supertype "Entity"  [override]
        def get(cls, pk=None, label=None, machinename=None):
        ^
src/aiida/orm/nodes/data/code/legacy.py:234: note:      Superclass:
src/aiida/orm/nodes/data/code/legacy.py:234: note:          @classmethod
src/aiida/orm/nodes/data/code/legacy.py:234: note:          def get(cls, **kwargs: Any) -> Any
src/aiida/orm/nodes/data/code/legacy.py:234: note:      Subclass:
src/aiida/orm/nodes/data/code/legacy.py:234: note:          @classmethod
src/aiida/orm/nodes/data/code/legacy.py:234: note:          def get(cls, pk: Any = ..., label: Any = ..., machinename: Any = ...) -> Any
```

Reply:
Since you asked, my answer is because we did bad design by keep on doing too much bad inheritance. 
I'd just add an `ignore`, otherwise need to add an other function signature dispatch or change the signature of parent function which can cause unexpect things.